lib/r10k/action/deploy/environment.rb

```diff
@@ -224,8 +189,8 @@ def allowed_initialize_opts
                       'generate-types': :self,
                       'puppet-path': :self,
                       'puppet-conf': :self,
-                      'sshkey-path': :self,
-                      'token-path': :self,
+                      'private-key': :self,
+                      'oauth-token': :self,
```

Comment:
ugh I wish these could still be `path`s, but I understand why we're doing this 😞